packages/@react-spectrum/tabs/docs/Tabs.mdx

```diff
@@ -0,0 +1,459 @@
+         {item => (
+          <Item title={item.name}>
+            <Content marginTop="size-250" marginStart="size-125">
+              <Text>
+                {item.children}
+              </Text>
+            </Content>
+          </Item>
+        )}
+      </Tabs>
+    </Flex>
+  );
+}
+```
+
+## Events
+Tabs supports selection via mouse, keyboard, and touch. You can handle all of these via the `onSelectionChange` prop. Tabs will pass the selected `key` to the `onSelectionChange` handler.
+
+The following example uses an `onSelectionChange` handler to update the tab selection stored in React state.
+
+```tsx example
+function Example() {
+  let tabs = [
+    {name: 'Triassic', children: 'The Triassic ranges roughly from 252 million to 201 million years ago, preceding the Jurassic Period.'},
+    {name: 'Jurassic', children: 'The Jurassic ranges from 200 million years to 145 million years ago.'},
+    {name: 'Cretaceous', children: 'The Cretaceous is the longest period of the Mesozoic, spanning from 145 million to 66 years ago.'}
+  ];
+  let [timePeriod, setTimePeriod] = React.useState('Triassic');
+
+  return (
+    <>
+      <p>Selected time period: {timePeriod}</p>
+      <Tabs aria-label="Mesozoic time periods" items={tabs} selectedKey={timePeriod} onSelectionChange={setTimePeriod}>
+        {item => (
+          <Item title={item.name} key={item.name}>
+            <Content marginTop="size-250" marginStart="size-125">
+              <Text>
+                {item.children}
+              </Text>
+            </Content>
+          </Item>
+        )}
+      </Tabs>
+    </>
+  );
+}
+```
+
+### Keyboard Activation
+By default, pressing the arrow keys while focus is on a Tab will switch selection to the adjacent Tab in that direction, updating the content displayed accordingly. If you would like to prevent selection change
+from happening automatically you can set the `keyboardActivation` prop to "manual". This will prevent tab selection from changing on arrow key press, requiring a subsequent `Enter` or `Space` key press to confirm
+tab selection.
+
+```tsx example
+function Example() {
+  let tabs = [
+    {name: 'Triassic', children: 'The Triassic ranges roughly from 252 million to 201 million years ago, preceding the Jurassic Period.'},
+    {name: 'Jurassic', children: 'The Jurassic ranges from 200 million years to 145 million years ago.'},
+    {name: 'Cretaceous', children: 'The Cretaceous is the longest period of the Mesozoic, spanning from 145 million to 66 years ago.'}
+  ];
+
+  return (
+    <Tabs aria-label="Mesozoic time periods" items={tabs} keyboardActivation="manual">
+      {item => (
+        <Item title={item.name} key={item.name}>
+          <Content marginTop="size-250" marginStart="size-125">
+            <Text>
+              {item.children}
+            </Text>
+          </Content>
+        </Item>
+      )}
+    </Tabs>
+  );
+}
+```
+
+## Complex Items
```

Comment:
This way of adding content to a title prop I think should be handled differently via API. Devon and I have discussed how to better handle complex Tab titles (maybe using children) but haven't come up with anything concrete. I think until we put some more thought into how we manage it, having this section seems premature. I would suggest removing Complex Items so that we do not advertise a particular method. 
Thoughts?

Reply:
That sounds fine, I'll go ahead and do that